dev/sparktestsupport/modules.py

```diff
@@ -389,19 +389,19 @@ def __hash__(self):
         "python/pyspark/sql"
     ],
     python_test_goals=[
-        "pyspark.sql.types",
-        "pyspark.sql.context",
-        "pyspark.sql.session",
-        "pyspark.sql.conf",
-        "pyspark.sql.catalog",
-        "pyspark.sql.column",
-        "pyspark.sql.dataframe",
-        "pyspark.sql.group",
-        "pyspark.sql.functions",
-        "pyspark.sql.readwriter",
-        "pyspark.sql.streaming",
-        "pyspark.sql.udf",
-        "pyspark.sql.window",
+        #        "pyspark.sql.types",
+        #        "pyspark.sql.context",
+        #        "pyspark.sql.session",
+        #        "pyspark.sql.conf",
+        #        "pyspark.sql.catalog",
+        #        "pyspark.sql.column",
+        #        "pyspark.sql.dataframe",
+        #        "pyspark.sql.group",
+        #        "pyspark.sql.functions",
+        #        "pyspark.sql.readwriter",
+        #        "pyspark.sql.streaming",
+        #        "pyspark.sql.udf",
+        #        "pyspark.sql.window",
```

Comment:
temp changes only to speed up local testing. will revert after first round of review.